datashader/dask.py

```diff
@@ -71,7 +71,7 @@ def chunk(df):
     dask_dataframe_keys = getattr(df, '__dask_keys__', None) or df._keys
     keys = dask_dataframe_keys()
     keys2 = [(name, i) for i in range(len(keys))]
-    dsk = dict((k2, (chunk, k)) for (k2, k) in zip(keys2, keys))
+    dsk = {k2: (chunk, k) for k2, k in zip(keys2, keys)}
```

Comment:
This seems like a stylistic change, in which case please revert it, but if it's needed e.g. for python2 then it's fine.